src/main/scala/beam/router/BeamSkimmer.scala

```diff
@@ -382,13 +398,66 @@ class BeamSkimmer @Inject()(val beamConfig: BeamConfig) extends IterationEndsLis
       event.getServices.getIterationNumber,
       BeamSkimmer.fullSkimsFileBaseName + ".csv.gz"
     )
-    val uniqueModes = skims.map(keyVal => keyVal._1._2).toList.distinct
+    val uniqueModes = skims.map(keyVal => keyVal._1._2).toList.distinct.filter(_.toString == "CAR")
     val uniqueTimeBins = (0 to 23)
 
     val dummyId = Id.create("NA", classOf[BeamVehicleType])
+
+    /*
     val writer = IOUtils.getBufferedWriter(filePath)
     writer.write(fileHeader)
     writer.write("\n")
+    */
+
+    val filePathObservedVsSimulated = event.getServices.getControlerIO.getIterationFilename(
+      event.getServices.getIterationNumber,
+      "tazODTravelTimeObservedVsSimulated.csv.gz"
+    )
+    val writerObservedVsSimulated = IOUtils.getBufferedWriter(filePathObservedVsSimulated)
+    writerObservedVsSimulated.write("fromTAZId,toTAZId,hour,timeSimulated,timeObserved")
+    writerObservedVsSimulated.write("\n")
+
+    val fjson = new FeatureJSON
+    val file = new File(beamConfig.beam.calibration.roadNetwork.travelTimes.benchmarkFileCensustracts)
+    val fileIn = new FileInputStream(file)
+    val features = fjson.readFeatureCollection(fileIn).features()
+
+    val trie: TrieMap[Path, Float] = TrieMap.empty
+    val series: XYSeries = new XYSeries("Time")
+
+    if (beamConfig.beam.calibration.roadNetwork.travelTimes.benchmarkFileCensustracts.nonEmpty &&
```

Comment:
Lines(L428-L460) could be extracted to the separate method to keep original method clean and small